### PR TITLE
V2022.10.0 rc

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR shows the differences between the v2022.10.0 RC branch and develop.
- No need to approve this as it will not be merged.
- **Please review this one (much easier) and approve this one** https://github.com/LLNL/RAJA/pull/1353

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -20,6 +20,89 @@ Notable changes include:
...
+       back-end reduction machinery directly, for OpenMP and SYCL for example.
+       The interface also enables user-chosen kernel names to be passed to
+       RAJA::forall for performance analysis annotations that are easier to
+       understand. Example codes are included as well as a description of
+       the new interface and comparison with the pre-existing interface in
+       the RAJA User Guide.
+     * Added support for run time execution policy selection for RAJA::forall
+       kernels. Users can specify any number of execution policies in their
+       code and then select which to use at run time. There is no discussion 
+       of this in the RAJA USer Guide yet. However, there are a couple of
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:49]:
```suggestion
       of this in the RAJA User Guide yet. However, there are a couple of 
```

File: examples/tut_launch_basic.cpp [hunk 2]
```diff
@@ -133,7 +133,11 @@ __global__ void gpuKernel()
 }
 #endif
 
-int main(int argc, char *argv[])
+#if defined(RAJA_ENABLE_CUDA) || defined(RAJA_ENABLE_HIP)
+int main(int argc, char* argv[])
+#else
+int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
```
Comment (MrBurmark) [inline examples/tut_launch_basic.cpp:139]:
```suggestion
int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv))
```

File: Dockerfile [hunk 3]
```diff
@@ -88,13 +88,13 @@ RUN . /opt/spack/share/spack/setup-env.sh && spack load cuda && \
     cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_COMPILER=g++ -DENABLE_CUDA=On -DCMAKE_CUDA_STANDARD=14 -DCMAKE_CUDA_ARCHITECTURES=70 -DENABLE_OPENMP=On .. && \
     make -j 4
 
-FROM ghcr.io/rse-ops/hip-ubuntu-20.04:hip-4.3.1 AS hip
+FROM ghcr.io/rse-ops/hip-ubuntu-20.04:hip-5.1.3 AS hip
```
Comment (trws) [inline Dockerfile:91]: Do we want to stick with 5.1 over 5.2 or 5.3 for this release?

File: examples/tut_launch_basic.cpp (hunk 2 above)
Comment (rhornung67) [reply examples/tut_launch_basic.cpp:139]: Fixed.

File: Dockerfile (hunk 3 above)
Comment (rhornung67) [reply Dockerfile:91]: 5.1.3 is the newest image in the rse-ops repo

**Review by MrBurmark — APPROVED** [review]

**Pull request closed without merging**